components/poll.js

```diff
@@ -103,7 +103,7 @@ function PollResult ({ v, progress }) {
   )
 }
 
-export function usePollVote ({ query = POLL_VOTE, itemId }) {
+export function usePollVote ({ query = POLL_VOTE, itemId, ...options }) {
```

Comment:
Is `...options` needed? I don't see `usePollVote` called with anything else than `query` or `itemId`.